### PR TITLE
Adding Porch documentation to the docs.nephio.org

### PR DESCRIPTION
**Pull request by CsatariGergely**
This pr adds the markdown documentation of Porch to docs.nephio.org . 
Open issues are collected in [here](https://github.com/nephio-project/nephio/issues/744#issuecomment-2194127158).

Fixes https://github.com/nephio-project/nephio/issues/744

### CODE REVIEW

File: content/en/docs/porch/_index.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+Short for Package Orchestration.
+
+## Porch in the Nephio architecture, history and outlook
+
+Porch is a key component of the Nephio architecture what was originally developed in the
+[kpt](https://github.com/kptdev/kpt) project. When kpt was donated to the [CNCF](https://www.cncf.io/projects/kpt/) it
+was decided that Porch will not be part of the code kpt project and the code was donated to Nephio.
+
+Porch is now mainteined by the Nephio community and it is a stable part of the Nephio R3 architecture, however there is
```
Comment (liamfallon) [inline content/en/docs/porch/_index.md:21]:
```suggestion
Porch is now mainteined by the Nephio community and it is a stable part of the Nephio R3 architecture. However there is
```

File: content/en/docs/porch/_index.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+an active disucssion about the future of the project. It is possible that Porch will be replaced in the Nephio
```
Comment (liamfallon) [inline content/en/docs/porch/_index.md:22]:
```suggestion
an active disucssion about the future of the project. It is possible that the current Porch component will be replaced in the Nephio
```

File: content/en/docs/porch/_index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+architecture with a different codebase implementing the same concepts but not in a backward compatible way. Potential
+candidates, like [pkgserver](https://docs.pkgserver.dev/) are discussed in the Nephio community.
```
Comment (liamfallon) [inline content/en/docs/porch/_index.md:24]:
```suggestion
candidates such as [pkgserver](https://docs.pkgserver.dev/) are being discussed in the Nephio community. 
```

**Review by liamfallon — COMMENTED** [review]
This is great @CsatariGergely , just some small wording comments.

**Comment by CsatariGergely** [comment]
@liamfallon thanks for the suggestions, I've applied them.

**Comment by liamfallon** [comment]
@CsatariGergely Apologies, I see you have do-not-merge-hold on this, I should not have LGTMd it. Anyway, it didn't go through.

**Comment by CsatariGergely** [comment]
@liamfallon now the rest of the tings were also added by @lovesprung  and myself. 

Ready for review. 

/assign @liamfallon @ciaranjohnston

**Review by liamfallon — COMMENTED** [review]
/approve

**Pull request merged**